source/extensions/tracers/xray/util.cc

```diff
@@ -0,0 +1,50 @@
+#include "extensions/tracers/xray/util.h"
+
+#include "absl/strings/ascii.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace XRay {
+
+bool wildcardMatch(absl::string_view pattern, absl::string_view input) {
```

Comment:
can you add a TODO for fuzzing? there's a lot of C level string hacking below.

Reply:
Sure.